neo/io/plexonio.py

```diff
@@ -241,17 +241,18 @@ def read_segment(self, lazy=False, cascade=True, load_spike_waveform=True):
         for chan, h in iteritems(eventHeaders):
             if lazy:
                 times = []
+                labels = None
```

Comment:
But inside the `EventArray` constructor is the following:

```python
if labels is None:
    labels = np.array([], dtype='S')
```

so the result of passing `labels=None` is that `labels` becomes an empty array of strings.